model_analyzer/config/generate/quick_plus_concurrency_sweep_run_config_generator.py

```diff
@@ -28,10 +27,8 @@
 from model_analyzer.config.generate.model_profile_spec import ModelProfileSpec
 from model_analyzer.result.result_manager import ResultManager
 from model_analyzer.result.run_config_measurement import RunConfigMeasurement
-from model_analyzer.result.run_config_result import RunConfigResult
 
 from model_analyzer.constants import LOGGER_NAME
-from model_analyzer.config.input.config_defaults import DEFAULT_RUN_CONFIG_MIN_CONCURRENCY, DEFAULT_RUN_CONFIG_MAX_CONCURRENCY
```

Comment:
@nv-braf This one is sketchy. It appears we hardcode this? Are we correctly honoring the values passed in?

Reply:
We get max concurrency directly from the config (see line 144). These are fine to be removed

Reply:
we don't need to honor a minimum?

Reply:
something like this?
https://github.com/triton-inference-server/model_analyzer/pull/654/commits/3612efa03d194f02abb9ac1492ed716abfc9d425

Reply:
We do honor the minimum in the quick search.  And, maybe we should for the concurrency sweep? But, I don't see the harm in providing these values to flesh out the detailed report for the user.